src/utils/upload/fileUploadUtils.js

```diff
@@ -247,30 +253,74 @@ const fileUploadUtils = {
         return { valid: validFiles, invalid: invalidFiles };
       };
 
-      const { valid: DGEUnfilteredFiles, invalid } = getFilesMatching('DGE_unfiltered', files);
-      const { valid: DGEFilteredFiles } = getFilesMatching('DGE_filtered', invalid);
+      const dgeUnfilteredFiles = getFilesMatching('DGE_unfiltered/', files);
+      const dgeFilteredFiles = getFilesMatching('DGE_filtered/', dgeUnfilteredFiles.invalid);
```

Comment:
why do we need to pass `dgeUnfilteredFiles.invalid` as an argument for `getFilesMatching` in here?